lib/ros/be/application/services/templates/jobs/kafka-connect/connector-provision-job.yml.erb

```diff
@@ -0,0 +1,52 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: kafka-connect-connector-provision
+spec:
+  template:
+    metadata:
+      name: kafka-connect-connector-provision
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      restartPolicy: OnFailure
+      initContainers:
```

Comment:
@kostyaplis Why are we using initContainers here?

Reply:
I understand that using initContainers allows us to run multiple jobs sequentially. I.e. in case of multiple connectors to be configured.
I've tested a template specifying multiple connectors with different settings, it been correctly  rendered to multiple init containers config and those were launched in sequence then....

Reply:
@duanshiqiang that's where i got it from: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/#init-containers-in-use